src/main/java/org/apache/ibatis/jdbc/ScriptRunner.java

```diff
@@ -186,11 +186,12 @@ private void checkForMissingLineTerminator(StringBuilder command) {
 
   private StringBuilder handleLine(StringBuilder command, String line) throws SQLException, UnsupportedEncodingException {
     String trimmedLine = line.trim();
-    if (trimmedLine.toUpperCase().startsWith("DELIMITER")) {
-        delimiter = trimmedLine.substring(10).trim();
-        return command;
-    }
```

Comment:
That code was not removed but replaced. See the discussion here: https://github.com/mybatis/mybatis-3/issues/355#issuecomment-76972167

You can use this syntax instead:
> --@DELIMITER /
> CREATE TRIGGER (.... );
> END;
> /
> --@DELIMITER ;

Reason 1: "DELIMITER" is not a stqndard SQL command, we need to escape/comment it with '--'
Reason 2: To comply with the MyBatis commands we use an additional '@' to differentiate commands from common comments